python/graphstorm/gsf.py

```diff
@@ -88,6 +88,14 @@
 from .dataloading import (GSgnnLinkPredictionTestDataLoader,
                           GSgnnLinkPredictionJointTestDataLoader,
                           GSgnnLinkPredictionPredefinedTestDataLoader)
+from .trainer import (GSgnnLinkPredictionTrainer,
```

Comment:
Why the Trainers and Inferrers are imported by gsf.py?
Are we using these classes?